### PR TITLE
Unique, unique together and relational checkings. String size checkings.

### PR DESCRIPTION
**Pull request by cromat**

### CODE REVIEW

File: django_seed/guessers.py [hunk 1]
```diff
@@ -15,40 +15,40 @@ def _timezone_format(value):
     :param value: The datetime value
     :return: A locale aware datetime
     """
-    if getattr(settings, 'USE_TZ', False):
-        return timezone.make_aware(value, timezone.get_current_timezone(), is_dst=False)
-    return value
+    return timezone.make_aware(value, timezone.get_current_timezone()) if getattr(settings, 'USE_TZ', False) else value
```
Comment (AngelOnFira) [inline django_seed/guessers.py:18]: This seems like quite a long line to change to

**Comment by cromat** [comment]
@AngelOnFira I had plans to get this working but because of lack of time it just got lower on my priority list.

**Comment by AngelOnFira** [comment]
@cromat No worries, I'll take care of it :)

**Comment by cromat** [comment]
@AngelOnFira Thank you :)

**Comment by AngelOnFira** [comment]
@cromat Any reason you closed this? I believe last time I looked at it, it still needed to be implemented. Either way, I'll just reopen it once I get around to it.

**Comment by cromat** [comment]
Hi, sorry. I went a little bit trough commits and thougt that you imolemented it on your way. If this is still valid, I will try to fix this in the evening to pass the tests 😁

**Comment by Dzordzu** [comment]
Any updates on this?

**Comment by AngelOnFira** [comment]
@Dzordzu see #82

**Pull request closed without merging**